py_wtf/cli/__init__.py

```diff
@@ -77,6 +77,10 @@ def index_dir_cmd(dir: str) -> None:
 
 
 def index_dir(dir: Path) -> Iterable[Module]:
+    # TODO: do something with .pyi files
+    # If there's a .pyi file with no corresponding .py -> just index .pyi
```

Comment:
When would that normally happen? Something something FFI?

Reply:
Yes, something something FFI or generated code or 🧙